test/client/fixtures/sandbox/node/methods-test.js

```diff
@@ -367,6 +367,18 @@ if (window.DOMParser && !browserUtils.isIE9) {
     });
 }
 
+if (Object.assign) {
+    test('Object.assign (GH-1208)', function () {
+        var iframe = document.createElement('iframe');
+
+        Object.assign(iframe, { src: '/iframe' });
+
+        strictEqual(iframe.src, urlUtils.getProxyUrl('/iframe', {
+            resourceType: urlUtils.stringifyResourceType({ isIframe: true })
+        }));
+    });
```

Comment:
Add more testcases:
- check returned value of the `Object.assign` 
- all cases from https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
- target is null, primitive type, function